src/streams.c

```diff
@@ -866,7 +814,7 @@ static Obj PRINT_OR_APPEND_TO_STREAM(Obj args, int append)
             else if ( IS_STRING_REP(arg) ) {
                 PrintString1(arg);
             }
-            else if ( TNUM_OBJ( arg ) == T_FUNCTION ) {
+            else if ( TNUM_OBJ(arg) == T_FUNCTION ) {
```

Comment:
If you change the formatting, then go the full way. As it is, I'd either remove this change, or go directly to using `IS_FUNC`, and clang-formatting.

Reply:
I figured out where this diff comes from. I re-used the code from `PRINT_OR_APPEND_TO_FILE`. Now github matches this to the line in `PRINT_OR_APPEND_TO_STREAM` (which is slightly different) since it shows the diff over two commits.

I guess I'll use git clang-format on the commit then.